README.md

```diff
@@ -14,11 +14,11 @@ Looking for typescript? Check out [graphqlgen](https://github.com/prisma/graphql
 | Boilerplate | less | more | more | some |
 | Docs | [docs](https://gqlgen.com) & [examples](https://github.com/99designs/gqlgen/tree/master/example) | [examples](https://github.com/graph-gophers/graphql-go/tree/master/example/starwars) | [examples](https://github.com/graphql-go/graphql/tree/master/examples) | [examples](https://github.com/samsarahq/thunder/tree/master/example)|
 | Query | :+1: | :+1: | :+1: | :+1: |
-| Mutation | :+1: | :construction: [pr](https://github.com/graph-gophers/graphql-go/pull/182) | :+1: | :+1: |
-| Subscription | :+1: | :construction: [pr](https://github.com/graph-gophers/graphql-go/pull/132) | :+1: | :+1: |
+| Mutation | :+1: | :+1: | :+1: | :+1: |
```

Comment:
This one hasn't merged yet, but it looks active.

Reply:
@vektah not sure why that PR is being singled out. `graph-gophers/graphql-go` supports mutations and what I understand of that PR is that it is an enhancement to the way mutation resolvers are implemented.

Reply:
Because it prevents a pretty common use case

Reply:
i.e. "GraphQL allows you to have a mutation and a query with the same name, but graph-gophers/graphql-go does not."?

Reply:
Yeah in graphql they (query, mutation, subscription) are all separate types, but there it has been merged them into one and just hopes they don't collide. 

I think it's pretty important to draw attention to when deciding which library to use, because you won't hit it until you are well into building the server and swapping at that point becomes pretty costly.

Reply:
That makes sense, thanks for clarifying. I'll remove this change from the PR.